### PR TITLE
Name filtering and output format for the components command

### PR DESCRIPTION
**Pull request by pkedy**
# Description

Adds name filtering and output formatting for the components command.

@artursouza You might be the best person to review this since you did #549.

## Issue reference

Resolves: #553

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by pkedy** [comment]
@artursouza could you review this?

File: pkg/kubernetes/components.go [hunk 1]
```diff
@@ -22,20 +27,51 @@ type ComponentsOutput struct {
...
-		return nil, err
+	filtered := []v1alpha1.Component{}
+	filteredSpecs := []configurationDetailedOutput{}
+	for _, c := range confs.Items {
+		confName := c.GetName()
+		if confName == "daprsystem" {
+			continue
+		}
+
+		if name == "" || confName == name {
```
Comment (artursouza) [inline pkg/kubernetes/components.go:56]: case insensitive?

**Review by artursouza — CHANGES_REQUESTED** [review]
Just one small comment.

File: pkg/kubernetes/components.go (hunk 1 above)
Comment (pkedy) [reply pkg/kubernetes/components.go:56]: Yeah, I'll apply that across the others too.

**Review by artursouza — APPROVED** [review]

**Pull request merged**